domain/src/main/java/com/example/domain/repositories/IRequestTokenRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.domain.repositories
+
+import kotlinx.coroutines.Deferred
+
+interface IRequestTokenRepository {
```

Comment:
Not critical, but we usually don't use the letter I in the beginning of the interface name. 
It was popular a few years ago, but then gone.